src/eom-preferences-dialog.c

```diff
@@ -318,6 +322,13 @@ eom_preferences_dialog_init (EomPreferencesDialog *pref_dlg)
 	                 G_SETTINGS_BIND_DEFAULT);
 
 	gtk_widget_show_all (priv->plugin_manager);
+
+	/* Add tab scrolling support for GTK3 */
+	gtk_widget_add_events (priv->notebook, GDK_SCROLL_MASK);
+	g_signal_connect (priv->notebook,
+					  "scroll-event",
+					  G_CALLBACK (eom_util_dialog_page_scroll_event_cb),
+					  NULL);
```

Comment:
fix code formatting

Reply:
There were 2 spaces, the size between github and this screenshot are different:
![Capture du 2020-05-19 11-02-52](https://user-images.githubusercontent.com/50292710/82307443-a50bb580-99c0-11ea-9bbb-0313945d05ae.png)

Last line with NULL with space in place to show you my problem.
I will correct in the next commit...

Thanks you very much for your help.